rubocop-airbnb/lib/rubocop/cop/airbnb/continuation_slash.rb

```diff
@@ -14,9 +14,7 @@ def enforce_violation(node)
         alias on_send enforce_violation
         alias on_if enforce_violation
 
-        rubocop_version = Gem.loaded_specs.fetch('rubocop', nil)&.version.to_s
-        assignments = rubocop_version.to_f >= 0.59 ? AST::Node::ASSIGNMENTS : Util::ASGN_NODES
```

Comment:
We lost this customization, but I can't see why this would be wrong in the master branch. Gonna have to ask John to figure out the history behind this one... 🤔